lib/items.js

```diff
@@ -28,6 +28,7 @@ const SCHEMA = joi.object().keys({
   origins: joi.array().items(STRING_500).max(5).default([]),
   tags: joi.array().items(STRING_500).max(10).default([]),
   entry: joi.alternatives(ENTRY_SCHEMAS).required(),
+  last_used: joi.date().allow(null).default(null),
```

Comment:
I'm not sure this is what we really want.  Adding it to the schema allows the API user to define when `last_used` was, including removing it altogether (accidentally or otherwise).

Other read-only properties (e.g., `modified`, `created`, `disabled`) are handled as a separate step post Joi validation.  That feels like the right behavior for this property, but I'm open to different opinions.